### DISCUSSION TITLE
Project extension

### DISCUSSION THREAD

**Discussion by wolfie7679** [SOEN345-WINTER2024/discussion#16] [General]
Hello,

I would like to request an extension as our workload is quite high with other projects.

Further, lab 9 is supposed to be today, and part of the project is about lab 9.

For some of us, lab 9 is after 5, at school, letting us not much time for finalizing everything

Thanks

**Comment by NicholasWahome** [comment]
Hello, 

I agree! 
I have a similar concern regarding the project. 
This would give me more time to give quality work.
Too many assignments at the moment

Thank you

**Comment by stan6** [comment]
Hi, as repeated in Progress Report, no extension will be provided.
